src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/Namespace/GetAzureServiceBusNamespaceKey.cs

```diff
@@ -13,14 +13,17 @@
 // ----------------------------------------------------------------------------------
 
 using Microsoft.Azure.Management.ServiceBus.Models;
+using Microsoft.Azure.Commands.ServiceBus.Models;
 using System.Management.Automation;
+using System;
 
 namespace Microsoft.Azure.Commands.ServiceBus.Commands.Namespace
 {
     /// <summary>
     /// 'Get-AzureRmServiceBusNamespaceKey' Cmdlet gives key detials for the given ServiceBus Namespace Authorization Rule
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, ServiceBusNamespaceKeyVerb), OutputType(typeof(ResourceListKeys))]
+    [ObsoleteAttribute("'Get-AzureRmServiceBusNamespaceKey' cmdlet is mark as obsolete and will be depricated in upcoming breaking changes build. Please use the New cmdlet 'Get-AzureRmServiceBusKey'", false)]
+    [Cmdlet(VerbsCommon.Get, ServiceBusNamespaceKeyVerb), OutputType(typeof(ListKeysAttributes))]
```

Comment:
@v-Ajnava shouldn't this be `AccessKeys`? According to the [SDK changes](https://github.com/Azure/azure-sdk-for-net/pull/3423/files#diff-7167456a2a74a724a6f204853bcf31a2L20) this was the type name change